### PR TITLE
fix: Remove focus within from buttons and fix keyboard navigation

### PR DESCRIPTION
**Pull request by samwhosung**
### Summary

Every time you click a button it focuses, I think this was a leftover when we where still wrapping inputs with the button component (which we are no doing as far a I see). 

Fixes keyboard navigation for button, before outline was disabled everywhere and now it's just disabled for inputs. Also added some buttons where I found them to be missing.

### CODE REVIEW

File: src/views/Editor.vue [hunk 1]
```diff
@@ -294,18 +294,12 @@ export default defineComponent({
...
-          text="Preview"
-          class="border-transparent"
-          @click="previewEnabled = true"
-        />
+        <button type="button" @click="previewEnabled = false">
+          <Link :is-active="!previewEnabled" text="Write" class="border-transparent" />
+        </button>
+        <button type="button" @click="previewEnabled = true">
+          <Link :is-active="previewEnabled" text="Preview" class="border-transparent" />
+        </button>
```
[the remark this answers is not in the document]
Comment (samuveth) [reply src/views/Editor.vue:302]: Yeah had a similar thought. Link should be more like this I think what we have on Snapshot https://github.com/snapshot-labs/snapshot/blob/master/src/components/BaseLink.vue

[the remark this answers is not in the document]
Comment (samuveth) [reply src/views/Editor.vue:302]: I added the buttons in Treasury as well

[the remark this answers is not in the document]
Comment (Sekhmet) [reply src/views/Editor.vue:302]: https://github.com/snapshot-labs/sx-ui/issues/873

**Review by Sekhmet — APPROVED** [review]
tACK

**Pull request merged**